apps/src/applab/applab.js

```diff
@@ -1879,6 +1888,27 @@ Applab.onDesignModeClear = function() {
   document.getElementById('divApplab').innerHTML = Applab.levelHtml = "";
 };
 
+Applab.onDesignModeManageAssets = function(assetChosen, typeFilter) {
+  var codeDiv = document.createElement('div');
+  var showChoseImageButton = assetChosen && typeof assetChosen === 'function';
+  var dialog = studioApp.createModalDialog({
+    Dialog: Dialog,
```

Comment:
Doesn't seem like we should need to pass Dialog into studioApp.createModalDialog (we should instead let it use it's copy of this.Dialog). This would mean applab.js no longer has an assumption that Dialog is a global